docs/sample_sheet_schema/schema_specification.md

```diff
@@ -39,7 +40,7 @@ As such, for CSV sample sheets, the top-level schema should be specified as an a
 ```
 
 If your sample sheet has a different format (for example, a YAML file with a nested structure),
-you will need to build your schema to match the parsed structure.
+you will need to build your schema to match the parsed structure. Using nested YAML files is (for now) not officially supported by the plugin and can result in unexpected errors.
```

Comment:
Didn't we just say that it was supported, a few lines above?

Reply:
Good point I'll remove that!